src/app/modals/personal-info/personal-info.component.ts

```diff
@@ -44,12 +39,16 @@ export class PersonalInfoComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.userService.currentUser.subscribe(user => this.currentUser = user);
+    this.userService.currentUser.subscribe((user: User) => this.currentUser = user);
     if (typeof (this.currentUser.birthday) === 'string') {
```

Comment:
có thể  thay 2 dòng này thành 1 dòng như sau
 if (typeof (this.currentUser.birthday) === 'string' && this.currentUser.birthday !== null) {
      this.currentUser.birthday = new Date(this.currentUser.birthday);
    }

Reply:
đã thay dòng đó rồi nhé!